src/Microsoft.VisualStudio.AppDesigner/Microsoft.VisualStudio.AppDesigner.vbproj

```diff
@@ -12,6 +12,9 @@
     <Summary>AppDesigner designer host implementation for Visual Studio project property pages</Summary>
     <PackageTags>Roslyn Project AppDesigner VisualStudio</PackageTags>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Remove="Microsoft.VisualStudio.ProjectSystem*" />
+  </ItemGroup>
```

Comment:
One of the issues I'll be making is about cleaning up our props/targets structure. While this works well enough, it is more like a bandaid on the props that this project includes. I'm just mentioning this here as an FYI.